config/main.py

```diff
@@ -623,12 +640,7 @@ def _restart_services():
         'telemetry'
     ]
 
-    disable_services = _get_disabled_services_list()
-
-    for service in disable_services:
-        if service in services_to_restart:
-            services_to_restart.remove(service)
-
+    _remove_invalid_services(services_to_restart)
```

Comment:
>_remove_invalid_services [](http://example.com/codeflow?start=4&length=24)

Please a testcase which fails old code and pass new code. It could be unit test or sonic-mgmt testcase.